### ISSUE TITLE
Unit test failure in  Validation/Geometry 

### ISSUE THREAD

**Issue by silviodonato** [cms-sw/cmssw#29814]
There is a unit test failure in Validation/Geometry  since CMSSW_11_1_X_2020-05-12-1100
```
[...]
Info in &lt;TCanvas::Print&gt;: png file Images/Tracker_x_over_l_vs_phi_Extended2017Plan1.png has been created
+ python /data/cmsbld/jenkins/workspace/ib-run-qa/CMSSW_11_1_X_2020-05-12-1100/src/Validation/Geometry/test/MaterialBudget.py -s -d Tracker -g Extended2015 -gc Extended2017Plan1
Error, missing file matbdg_Tracker_Extended2015.root
Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-qa/CMSSW_11_1_X_2020-05-12-1100/src/Validation/Geometry/test/MaterialBudget.py", line 1082, in &lt;module&gt;
    args.geometry_comparison)
  File "/data/cmsbld/jenkins/workspace/ib-run-qa/CMSSW_11_1_X_2020-05-12-1100/src/Validation/Geometry/test/MaterialBudget.py", line 181, in createCompoundPlotsGeometryComparison
    geometryOld,geometryNew)
  File "/data/cmsbld/jenkins/workspace/ib-run-qa/CMSSW_11_1_X_2020-05-12-1100/src/Validation/Geometry/test/MaterialBudget.py", line 59, in paramsGood_
    raise RuntimeError
RuntimeError

---&gt; test materialBudgetTrackerPlots had ERRORS
 
^^^^ End Test materialBudgetTrackerPlots ^^^^
&gt;&gt; Tests for package Validation/Geometry ran.
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_1_X_2020-05-12-1100/unitTestLogs/Validation/Geometry#/

Trying to debug step-by-step with
```
cmsRun test/single_neutrino_cfg.py nEvents=1000 
python test/runP_Tracker.py geom=Extended2015 label=Tracker 
12-May-2020 20:22:15 CEST  Initiating request to open file file:single_neutrino_random.root
12-May-2020 20:22:16 CEST  Successfully opened file file:single_neutrino_random.root
----- Begin Fatal Exception 12-May-2020 20:22:48 CEST-----------------------
An exception of category 'MuonSensitiveDetector' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'p1'
   [2] Calling method for module OscarMTProducer/'g4SimHits'
Exception Message:
Cannot find MuonGeometryConstants
----- End Fatal Exception -------------------------------------------------
12-May-2020 20:22:48 CEST  Closed file file:single_neutrino_random.root
```

This should be caused by #29632 (@bsunanda)

**Comment by silviodonato** [comment]
assign geometry

**Comment by silviodonato** [comment]
fixed by #29816

**Comment by cvuosalo** [comment]
+1
